test/SlowTests/Server/Documents/AI/AiAgent/AiAgentBasics.cs

```diff
@@ -55,15 +51,21 @@ public async Task CanCreateAiAgent(Options options, GenAiConfiguration config)
 
             agent.Queries =
             [
-                AiAgentConfiguration.ToolQuery.Build(
-                    "ProductSearch", 
-                    "semantic search the store product catalog",
-                    session.Query<Product>().VectorSearch(v => v.WithText(p => p.Name), v => v.ByText("$query")))
+                new AiAgentConfiguration.ToolQuery
+                {
+                    Name = "ProductSearch", 
+                    Description =  "semantic search the store product catalog",
+                    Query = "from Products where vector.search(embedding.text(Name), $query)",
+                    ParametersSchema = "{\"query\": [\"term or phrase to search in the catalog\"]}"
```

Comment:
I really dislike that this is a string - we need to think how we can make that an object.